core/src/main/java/org/infinispan/configuration/parsing/Parser.java

```diff
@@ -172,9 +172,11 @@ private void parseSerialization(final XMLExtendedStreamReader reader, final Conf
                parseSerializationContextInitializer(reader, holder.getClassLoader(), builder.serialization());
                break;
             }
-            case WHITE_LIST: {
+            case WHITE_LIST:
+               CONFIG.elementDeprecatedUseOther(Element.WHITE_LIST, Element.ALLOW_LIST);
```

Comment:
If the 12.0 schema is used shouldn't this thrown unrecognised attribute exception?

Reply:
Done